### PR TITLE
Revamping registry docs

### PR DESCRIPTION
**Pull request by mohab-sameh**

### CODE REVIEW

File: docs/codemod-registry/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: "Quickstart"
+hide_title: true
+---
+
+# Quickstart
+
+Codemod Registry is a collection of codemods and automation recipes built on top of popular engines like Meta’s jscodeshift, ts-morph, and less popular yet very powerful ones such as Uber's piranha.
+
+
+The Codemod Registry is integrated into Intuita's platform. Adding your codemods to the registry allows you to use them inside Intuita's powerful platform. Additionally, contributing to the Codemod Registry allows you to easily ship your codemods to developers and progressive teams around the world.
```
Comment (alex-from-intuita) [inline docs/codemod-registry/quickstart.md:11]: lets not call our platform powerful. no fluffs. just facts.
alternative to this paragraph:

Once codemods are added to this registry, they automatically get integrated into the rest of the Intuita platform, and anyone in the community can easily discover, run, and share them with just one click via Intuita's CLI or VS Code extension.

Comment (alex-from-intuita) [reply docs/codemod-registry/quickstart.md:11]: @mohab-sameh

File: docs/codemod-registry/importing-codemods.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+title: Importing Codemods
+---
+
+import ThemedImage from "@theme/ThemedImage";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import VideoSwitcher from '../../src/components/VideoSwitcher.tsx';
+
+# Importing Codemods Into the Intuita VS Code Extension
```
Comment (alex-from-intuita) [inline docs/codemod-registry/importing-codemods.md:11]: soon to talk about private registry (we will only talk about them once we have user accounts).

we should only talk about public registry in this page. 
@mohab-sameh

File: docs/codemod-registry/importing-codemods.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+1. Create a directory that will contain your Piranha rule.
+2. Place the Piranha rule inside a file called `rules.toml`.
+3. Right-click on the containing directory and click `Run as a Piranha Rule`.
+
+
+---
+
+## Importing codemods into the public registry
+
+Intuita ensures that contributions are of high quality and pass Intuita's guidelines and governance so that it is ready to be shared with the community.
```
Comment (alex-from-intuita) [inline docs/codemod-registry/importing-codemods.md:78]: some alternatives here:

"To ensure quality and usefulness of codemods and code automation recipes, Intuita provides guidelines and sets some basic governance rules. We'll also be building features to empower the community to uprank the highest quality codemods and recipes. Here is the process to add your codemods to the registry."

File: docs/codemod-registry/importing-codemods.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+2. [Open a PR with your codemod.](#opening-a-pr-with-your-codemod)
+
+
+### Understanding the registry's structure
+
+The codemod registry consists of many codemods running over various codemod engines (jscodeshift, ts-morph, Uber's Piranha, and Intuita's File Transformation Engine).
+
+The codemod registry contains directories for each framework/library. Within each directory, you can find the supported framework/library versions. For each version, you can find all the supported codemods.
+
+Codemod registry structure:
```
Comment (alex-from-intuita) [inline docs/codemod-registry/importing-codemods.md:92]: awesome.

File: docs/codemod-registry/importing-codemods.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+    ├── redwoodjs/
+    │   └── ...
+    └── ...
+```
+
+You can contribute to the registry by adding a completely new framework/library including supported codemods, or you can add a codemod to an existing framework/library.
+
+### Opening a PR with your codemod
+
+To contribute your codemod to the registry, you can [open a PR](https://github.com/intuita-inc/codemod-registry/pulls) containing your codemod. Your codemod should meet the following criteria:
```
Comment (alex-from-intuita) [inline docs/codemod-registry/importing-codemods.md:119]: show an example of a nice codemod metadata and readme file. @mohab-sameh

File: docs/codemod-registry/importing-codemods.md (hunk 2 above)
Comment (alex-from-intuita) [reply docs/codemod-registry/importing-codemods.md:11]: the title here should say:

Adding existing/new codemods to the Registry

File: docs/codemod-registry/quickstart.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Quickstart"
+hide_title: true
+---
+
+# Quickstart
```
Comment (alex-from-intuita) [inline docs/codemod-registry/quickstart.md:6]: quickstart does not make sense that much here. (there is nothing for them to do)
this is more of a intro page to registry.

we should convince folks why they should add their new/existing codemods to the registry.
we should highlight the benefits that i mentioned below so it becomes a no brainer for folks to add their codemods to the registry. @mohab-sameh

File: docs/codemod-registry/quickstart.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+
+Codemod Registry is a collection of codemods and automation recipes built on top of popular engines like Meta’s jscodeshift, ts-morph, and less popular yet very powerful ones such as Uber's piranha.
+
+
+The Codemod Registry is integrated into Intuita's platform. Adding your codemods to the registry allows you to use them inside Intuita's powerful platform. Additionally, contributing to the Codemod Registry allows you to easily ship your codemods to developers and progressive teams around the world.
+> 👩🏼‍💻 [Check out the Codemod Registry here →](https://github.com/intuita-inc/codemod-registry)
+
+
+## Codemod Registry vs. the Platform
```
Comment (alex-from-intuita) [inline docs/codemod-registry/quickstart.md:15]: better to highlight the registry so quickly folks understand where in the architecture they should look at.

File: docs/codemod-registry/quickstart.md (hunk 6 above)
Comment (mohab-sameh) [reply docs/codemod-registry/quickstart.md]: @alex-from-intuita renamed it to Intro. Will do for now?

Comment (mohab-sameh) [reply docs/codemod-registry/quickstart.md]: @alex-from-intuita renamed it to 'Introduction'. Will do for now?

**Review by alex-from-intuita — APPROVED** [review]

**Pull request merged**